tests/e2e/util.go

```diff
@@ -1068,13 +1068,20 @@ func getCNSRegisterVolumeSpec(ctx context.Context, namespace string, fcdID strin
 			Namespace:    namespace,
 		},
 		Spec: cnsregistervolumev1alpha1.CnsRegisterVolumeSpec{
-			PvcName:  persistentVolumeClaimName,
-			VolumeID: fcdID,
+			PvcName: persistentVolumeClaimName,
 			AccessMode: v1.PersistentVolumeAccessMode(
 				accessMode,
 			),
 		},
 	}
+
+	if vmdkPath != "" {
```

Comment:
is it ok if both of them are empty?

Reply:
Yes, it is Ok. It still creates cnsRegisterVolume , But It fails in creating PV PVC.  So , in waitForCNSRegisterVolumeToGetCreated I am verifying "Registered" status. If that is false it throws error and fails the test. That part is handled.